BUILD.gn

```diff
@@ -0,0 +1,530 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/features.gni")
+import("//build/config/sanitizers/sanitizers.gni")
+import("//build/config/ui.gni")
+import("//build_overrides/v8.gni")
+import("//media/media_options.gni")
+import("//third_party/widevine/cdm/widevine.gni")
+import("//third_party/ffmpeg/ffmpeg_options.gni")
+import("//xwalk/build/version.gni")
+
+config("xwalk_runtime_config") {
```

Comment:
This is a very generic name. In any case, we don't really support Precise anymore so I'll just remove this from gyp and you can drop it from the GN code too.